File: codewars/stop_spinning_my_words.py
"""
Write a function that takes in a string of one or more words, 
and returns the same string, but with all five or more letter words reversed
"""

def spin_words(sentence):
    # Your code goes here
    arr=sentence.split()
    ans=[]
    for i in arr:
        if len(i) >= 5:
            ans.append(i[::-1])
        else:
            ans.append(i)
    return ' '.join(ans)

File: codewars/stopSpinningMyWords.js
/*
Write a function that takes in a string of one or more words, 
and returns the same string, but with all five or more letter words reversed
*/

function spinWords(sentence){
    //converting stfring to array to get words
    let arr=sentence.split(" ");
    for(let i=0;i<arr.length;i++){
      //checking which of the words is equal to or more than 5 letters
      if(arr[i].length>=5){
        //reversing the words which satisfy the condition
        arr[i]=arr[i].split('').reverse().join('');
      }
    }
    //returning the formatted string
    return arr.join(' ');
  }